Items.cs

```diff
@@ -1,418 +1,430 @@
-        const int PFI_Maze_RR = 183;
-        const int PR_RR1 = 184;
-        const int PR_RR2 = 185;
-        const int Bomber_Hideout_SR = 186;
-        const int Termina_Bombchu_Grotto = 187;
-        const int Termina_RR_Grotto = 188;
-        const int Termina_Underwater_RR = 189;
-        const int Termina_Grass_RR = 190;
-        const int Termina_Stump_RR = 191;
-        const int GB_Coast_Grotto = 192;
-        const int GB_Cape_Ledge_1 = 193;
-        const int GB_Cape_Ledge_2 = 194;
-        const int GB_Cape_Grotto = 195;
-        const int GB_Cape_Underwater = 196;
-        const int PFE_RR_1 = 197;
-        const int PFE_RR_2 = 198;
-        const int PFE_RR_3 = 199;
-        const int To_Swamp_Grotto = 200;
-        const int Dog_Race_PR = 201;
-        const int Graveyard_Grotto = 202;
-        const int Swamp_Grotto = 203;
-        const int WF_BR = 204;
-        const int WF_RR = 205;
-        const int Well_Right_PR = 206;
-        const int Well_Left_PR = 207;
-        const int MV_Chest = 208;
-        const int MV_Grotto = 209;
-        const int To_Ikana_RR = 210;
-        const int To_Ikana_Grotto = 211;
-        const int IST_SR = 212;
-        const int IST_10Chu = 213;
-        const int IST_Bean = 214;
-        const int To_SH_Grotto = 215;
-        const int To_GV_RR = 216;
-        const int Secret_Shrine_HP = 217;
-        const int Secret_Shrine_Dino = 218;
-        const int Secret_Shrine_Wizz = 219;
-        const int Secret_Shrine_Wart = 220;
-        const int Secret_Shrine_Garo = 221;
-        const int Inn_Staff_Room = 222;
-        const int Inn_Guest_Room = 223;
-        const int Woods_Grotto = 224;
-        const int ECT_SR = 225;
-        const int SCT_RR = 226;
-        const int SCT_PR = 227;
-        const int Bank_HP = 228;
+        public const int ChestLensCaveRedRupee = 171;
+        public const int ChestLensCavePurpleRupee = 172;
+        public const int ChestBeanGrottoRedRupee = 173;
+        public const int ChestHotSpringGrottoRedRupee = 174;
+        public const int GrottoBadBats = 175;
+        public const int GrottoIkana = 176;
+        public const int ChestPiratesFortressRedRupee1 = 177;
+        public const int ChestPiratesFortressRedRupee2 = 178;
+        public const int ChestInsidePiratesFortressTankRedRupee = 179;
+        public const int ChestInsidePiratesFortressGuardSilverRupee = 180;
+        public const int ChestInsidePiratesFortressHeartPieceRoomRedRupee = 181;
+        public const int ChestInsidePiratesFortressHeartPieceRoomBlueRupee = 182;
+        public const int ChestInsidePiratesFortressMazeRedRupee = 183;
+        public const int ChestPinacleRockRedRupee1 = 184;
+        public const int ChestPinacleRockRedRupee2 = 185;
+        public const int ChestBomberHideoutSilverRupee = 186;
+        public const int GrottoTerminaBombchu = 187;
+        public const int Termina_RR_Grotto = 188; // TODO Grotto with red rupee? Grotto next to red rupee grotto?
```

Comment:
To answer your question: everything in the "other chests and grotto" section except Bank_HP is in a Chest.

Reply:
Are you trying to separate which chests are in grottos and which aren't?

Reply:
Oh, I see - thanks. Nah, I think I tried to derive the type of "item" it was from its previous name, when it didn't include chest or hp, I assumed it was specifying an actual grotto. Will include chest in the name for clarification.